integration_tests/nlg_meta_eval_test.py

```diff
@@ -39,3 +40,94 @@ def test_da_cs_en(self):
 
         self.assertGreater(len(sys_info.results.analyses), 0)
         self.assertGreater(len(sys_info.results.overall), 0)
+
+
+class MetaEvalNLGTest(unittest.TestCase):
+    true_data = [[1, 2, 3, 4, 5], [2, 1, 4, 5, 2], [5, 4, 3, 2, 1]]
+    pred_data = [[2, 1, 3, 4, 5], [2, 4, 5, 5, 2], [5, 3, 4, 2, 1]]
+
+    def test_sample_level_spearmanr(self):
```

Comment:
Add `-> None` annotation to all test functions so that the type checker captures these functions.

```suggestion
    def test_sample_level_spearmanr(self) -> None:
```